tests/de/gurkenlabs/litiengine/resources/FontsTests.java

```diff
@@ -0,0 +1,58 @@
+package de.gurkenlabs.litiengine.resources;
+
+import org.junit.jupiter.api.Test;
+
+import java.awt.*;
```

Comment:
That's a heavy wildcard import right there :o 
Please use explicit imports instead.

Reply:
Took away the heavy import and changed it to explicit.